@kiva/kv-components/src/vue/KvSelect.vue

```diff
@@ -112,3 +112,13 @@ export default {
 	},
 };
 </script>
+
+<style lang="postcss" scoped>
+.container {
+	@apply tw-relative tw-w-full;
+
+	@screen md {
+		max-width: 192px
```

Comment:
I don't think this should be applied globally... if there is a need for a max width we should probably do that on a context basis.

Also, why would we allow the select to be 300ish pixels on mobile but only 192 on tablet or desktop...

Reply:
The design reqs on this one was confusing, but I just re-read the comment thread, and you're correct that designers actually didn't accept this req across the board. Looks like I forgot to change the ticket based on the comment thread.

@christian14b I think we can apply the ellipsis globally, then have the max-width set in the combo page.

I'm also seeing that there are other changes in the Figma related to the dropdown changes. Slight rounded corner changes and whatnot. Those we can also do in this changeset.